frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-details.tsx

```diff
@@ -6,19 +6,65 @@ import {
   SectionHeading,
   ResourceSummary,
   Kebab,
+  DetailsItem,
+  useAccessReview,
 } from '@console/internal/components/utils';
 import { CatalogSourceListPage, CatalogSourceListPageProps } from '../catalog-source';
 import { OperatorHubKind } from '.';
 import { OperatorHubModel } from '../../models';
+import { editDefaultSourcesModal } from '../modals/edit-default-sources-modal';
 
-const OperatorHubDetails: React.FC<OperatorHubDetailsProps> = ({ obj }) => {
+const OperatorHubDetails: React.FC<OperatorHubDetailsProps> = ({ obj: operatorHub }) => {
   const { t } = useTranslation();
+
+  const canEditDefaultSources = useAccessReview({
+    group: OperatorHubModel.apiGroup,
+    resource: OperatorHubModel.plural,
+    verb: 'update',
```

Comment:
Should this be `patch` if the modal uses `k8sPatch`?

Reply:
I'm actually not really sure. I'm not intimately familiar with how the access review API works.

Reply:
Yeah, it should be patch if this action spawns the dialog.